src/utilities/toast-manager.test.ts

```diff
@@ -1,3 +1,92 @@
+import faker from "faker";
+import { ToastManager } from "./toast-manager";
+
 describe("ToastManager", () => {
-    test.skip("TODO: Backfill tests for issue https://github.com/AndcultureCode/AndcultureCode.JavaScript.React.Components/issues/34", () => {});
+    test("when error toast created, returns toastId", () => {
```

Comment:
We typically wrap each function being tested in its own `ts-region` block and a `describe` block. While these sections may only contain one test right now, it's more consistent with the rest of our test suites and will be easier to maintain if additional tests need to be added in here.

In terms of naming sequential tests, it could also reduce some of the boilerplate due to the context:

```
// -----------------------------------------------------------------------------------------
// #region info
// -----------------------------------------------------------------------------------------

describe("info", () => {
    test("when toastId provided, returns toastId", () => {
        // Arrange
        const expected = faker.random.number(99);
        const toastContent = faker.random.word();
        const toastOptions = { toastId: expected };
    
        // Act
        const toastId = ToastManager.info(toastContent, toastOptions);
    
        // Assert
        expect(toastId).toBe(expected);
    });
})

// #endregion info
```

I'd recommend a similar change for the toast template tests ☝️